digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -31,6 +34,12 @@
 public class TDOperator
         implements Closeable
 {
+    private static final Logger logger = LoggerFactory.getLogger(TDOperator.class);
+
+    private static final int INITIAL_RETRY_WAIT = 100;
```

Comment:
Per discussion on slack, will change this to two retries with 300 ms initial wait.